utilities-infrastructure-image-codebuild.tf

```diff
@@ -0,0 +1,144 @@
+  role       = aws_iam_role.infrastructure_utilities_image_codebuild[0].name
+  policy_arn = aws_iam_policy.infrastructure_utilities_image_codebuild_cloudwatch_rw[0].arn
+}
+
+resource "aws_iam_policy" "infrastructure_utilities_image_codebuild_allow_builds" {
+  count = local.enable_infrastructure_utilities ? 1 : 0
+
+  name        = "${local.resource_prefix}-${substr(sha512("utilities-image-codebuild-allow-builds"), 0, 6)}"
+  description = "${local.resource_prefix}-utilities-image-codebuild-allow-builds"
+  policy      = templatefile("${path.root}/policies/codebuild-allow-builds.json.tpl", {})
+}
+
+resource "aws_iam_role_policy_attachment" "infrastructure_utilities_image_codebuild_allow_builds" {
+  count = local.enable_infrastructure_utilities ? 1 : 0
+
+  role       = aws_iam_role.infrastructure_utilities_image_codebuild[0].name
+  policy_arn = aws_iam_policy.infrastructure_utilities_image_codebuild_allow_builds[0].arn
+}
+
+resource "aws_iam_policy" "infrastructure_utilities_image_codebuild_ecr_push" {
+  count = local.enable_infrastructure_utilities ? 1 : 0
+
+  name        = "${local.resource_prefix}-${substr(sha512("utilities-image-codebuild-ecr-push"), 0, 6)}"
+  description = "${local.resource_prefix}-ecs-service-codepipeline-codebuild-ecr-push"
+  policy = templatefile(
+    "${path.root}/policies/ecr-push.json.tpl",
+    { ecr_repository_arn = aws_ecr_repository.infrastructure_utilities[0].arn }
+  )
+}
+
+resource "aws_iam_role_policy_attachment" "infrastructure_utilities_image_codebuild_ecr_push" {
+  count = local.enable_infrastructure_utilities ? 1 : 0
+
+  role       = aws_iam_role.infrastructure_utilities_image_codebuild[0].name
+  policy_arn = aws_iam_policy.infrastructure_utilities_image_codebuild_ecr_push[0].arn
+}
+
+resource "aws_codebuild_project" "infrastructure_utilities_image_build" {
+  count = local.enable_infrastructure_utilities ? 1 : 0
+
+  name          = "${local.resource_prefix}-utilities-image-build"
+  description   = "${local.resource_prefix} Utilities Image Build"
+  build_timeout = "20"
+  service_role  = aws_iam_role.infrastructure_utilities_image_codebuild[0].arn
+
+  artifacts {
+    type = "NO_ARTIFACTS"
+  }
+
+  environment {
+    compute_type    = "BUILD_GENERAL1_SMALL"
+    image           = "aws/codebuild/standard:7.0"
+    type            = "LINUX_CONTAINER"
+    privileged_mode = true
+
+    environment_variable {
+      name  = "AWS_ACCOUNT_ID"
+      value = local.aws_account_id
+    }
+
+    environment_variable {
+      name  = "REPOSITORY_URI"
+      value = aws_ecr_repository.infrastructure_utilities[0].repository_url
+    }
+
+    environment_variable {
+      name  = "DOCKERHUB_USERNAME"
+      value = local.infrastructure_dockerhub_username
+    }
+
+    environment_variable {
+      name  = "DOCKERHUB_TOKEN"
+      value = local.infrastructure_dockerhub_token
+    }
+  }
+
+  source {
+    type            = "GITHUB"
+    location        = "https://github.com/dxw/dalmatian-sql-backup"
```

Comment:
is it time to rename the repo now?

Reply:
Not just yet - I'm going to want to update this bit on it's own (Just in case the redirect doesn't work)